test/fixtures/main.tf

```diff
@@ -14,18 +14,35 @@
  * limitations under the License.
  */
 
-locals {
-  credentials_file_path = var.credentials_file_path
-}
-
 provider "google" {
-  credentials = file(local.credentials_file_path)
+  version = "2.18.0"
 }
 
 module "datastore" {
-  source      = "../../"
-  credentials = local.credentials_file_path
-  project     = var.project
-  indexes     = file("${path.module}/yaml/index.yaml")
+  source  = "../../"
+  project = var.project_id
+  indexes = data.null_data_source.dependency.outputs.indexes
+}
+
+resource "google_app_engine_application" "app" {
```

Comment:
This can be removed since it's in the example now.

Reply:
Not really, as fixtures don't use example